core/test_blockchain.go

```diff
@@ -1003,17 +1003,20 @@ func TestReorgReInsert(t *testing.T, create func(db ethdb.Database, chainConfig
 	checkBlockChainState(t, blockchain, gspec, chainDB, create, checkState)
 }
 
-//nolint:goimports
 // Insert two different chains that result in the identical state root.
 // Once we accept one of the chains, we insert and accept A3 on top of the shared
 // state root
-//   G   (genesis)
-//  / \
+//
```

Comment:
nit: revert unrelated comment changes

Reply:
it may come from the code editor. I reverted 👍

Reply:
Thanks the linter can be really aggressive with these diagrams and we haven't found a good setting that keeps them easy to read and satisfies the linter.

If you're interested in learning/contributing more, it would be pretty easy to review and be good exposure to how the VM/engine interact. Documentation for the invariants between them is here: https://github.com/ava-labs/avalanchego/blob/master/vms/README.md

Reply:
Could you remove the added empty new lines, so that there is no unrelated diff as part of this PR?

Reply:
@aaronbuchwald Thank you for the link ! I start slowly to look at some easy issues on `avalanche` `subnet-evm` and `avalanche-cli`.
If you have some issues for new contributors don't hesitate to ping me.

Reply:
What's the best place to ping you?

Reply:
You can identify me on some issues, I should receive the notifications.
Otherwise if you have a valid discord invitation to Avalanche server it would be a good way too